Algo/alarm.js

```diff
@@ -0,0 +1,15 @@
+var prompt = require("prompt");
+var player = require("play-sound")((opts = {}));
+
+//
+// Start the prompt
+prompt.start();
+
+prompt.get(["timeToAlarm"], function(err, result) {
+  function setAlarm() {
+    player.play("BleedingOut.mp3", function(err) {
```

Comment:
looks like this might need a special local environment, would be helpful to add comments inline or add steps to a README